recipes/cpp-httplib/all/test_package/conanfile.py

```diff
@@ -1,17 +1,25 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import CMake, cmake_layout
 import os
 
 
 class TestPackageConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
-    generators = "cmake", "cmake_find_package_multi"
+    generators = "CMakeToolchain", "CMakeDeps", "VirtualRunEnv"
```

Comment:
Why `VirtualRunEnv`?
I'm sorry if this is a silly question, I'm trying to catch up with Conan 2.0. AFAIK we use this to get executable in PATH.

Reply:
Useless for header only libs, but mandatory if there is shared lib, specially on Windows since there is no rpath mechanism (to inject lib or bin folder of packages to `PATH`/`LD_LIBRARY_PATH`/`DYLD_LIBRARY_PATH` when you run `self.run(..., env="conanrun")` in `test()`). For consistency in CCI recipes, I keep it in all recipes.

Reply:
How does conan 1.x addresses this?

Reply:
with `run_environment=True` argument in `self.run()`